README.md

```diff
@@ -309,6 +309,30 @@ The following settings are optional and allow you to further configure your clus
 * `settings.kubernetes.standalone-mode`: Whether to run the kubelet in standalone mode, without connecting to an API server.  Defaults to `false`.
 * `settings.kubernetes.authentication-mode`: Which authentication method the kubelet should use to connect to the API server, and for incoming requests.  Defaults to `aws` for AWS variants, and `tls` for other variants.
 * `settings.kubernetes.bootstrap-token`: The token to use for [TLS bootstrapping](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/).  This is only used with the `tls` authentication mode, and is otherwise ignored.
+* `settings.kubernetes.eviction-hard`: The hard eviction Signal and Thresholds you set up to reclaim the associated starved resource.
+  Remember to quote keys (since they often contain ".") and to quote all values.
+  * Example user data for setting up eviction hard:
+    ```
+    [settings.kubernetes.eviction-hard]
+    "memory.available" = "15%"
+    "nodefs.inodesFree" = "15Mi"
+    "pid.available" = "15%"
+    ```
+* `settings.kubernetes.kube-reserved`: Resources reserved for node components.
+  * Example user data for setting up kube reserved:
+    ```
+    [settings.kubernetes.kube-reserved]
```

Comment:
I remembered one more thing, there was a not at at the of `kube-reserved` [doc](https://v1-18.docs.kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#kube-reserved). **Note that Kubelet does not create --kube-reserved-cgroup if it doesn't exist. Kubelet will fail if an invalid cgroup is specified**. We don't have to do anything for it ? Does that cgroup already exisit or how does it work ?

Reply:
I think for now we don't have to do anything with `--kube-reserved-cgroup`.  Because first I think Customers did not ask for that, so we do not need to provide that at this moment. Second, I went through how `amazon-eks-ami` deals with `kube-reserved` and `kube-reserved-cgroup`, and tried to find if that put some defaults for `kube-reserved-cgroup`. I could not find anything they did for that, so I assume it's an optional and not necessary at this moment. Does that make sense? Please correct me if I am wrong on this. Thanks 👍

Reply:
What you say makes sense, we may not need to set it. However, I have very limited knowledge here, I would let someone who understands relevance of cgroup here to comment.

Reply:
I'm not sure if I understand your concern. Are you concerned kubelet would break if we don't set it? We're currently not placing kubernetes-related daemons in the same cgroup so I don't think we need to set this at the moment. I don't think we would expose this as a setting even if we did (bottlerocket determines which cgroups different daemon falls under, not something to configure on the fly).

Reply:
I am not concerned, I just wanted to inform there was a Note: **Note that Kubelet does not create --kube-reserved-cgroup if it doesn't exist. Kubelet will fail if an invalid cgroup is specified.** in kubernetes doc. From what I understood form the note was that we would have to set the `kube-reserved-cgroup` for `kube-reserved` setting to work. 
However if that was not the case and if `kube-reserved` setting works without it then its all good. I just wanted to inform that this note was there and understand if we had to do anything for it.